pallets/settlement/src/lib.rs

```diff
@@ -1175,7 +1173,8 @@ impl<T: Trait> Module<T> {
                     }
                 }
             }
-            weight_for::weight_for_execute_instruction_if_no_pending_affirm::<T>(transaction_weight)
+            // TODO: Fix this
```

Comment:
Comment could be more specific. Not enough info for someone else to fix it later.